src/lib/components/AccessRightField.js

```diff
@@ -49,37 +47,49 @@ class Protection {
 }
 
 
-export class AccessRightField extends Component {
+function convertToNewFormat(access) {
+  let record;
+  if (access.record && ["public", "restricted"].includes(access.record)) {
```

Comment:
That first check is redundant as worst case the `access.record` will returned undefined which satisfies the second check I think...

Reply:
yes! And now that the backend is fine, I should be able to remove this alltogether.